src/components/ble/NotificationManager.cpp

```diff
@@ -44,9 +45,9 @@ NotificationManager::Notification NotificationManager::GetNext(NotificationManag
     result = *(notifications.begin());
   else
     result = *(currentIterator + 1);
-
+  
   if (result.id <= id)
-    return {};
+    return Notification{};
```

Comment:
just `return {};` should be enough, I think. No need to repeat the return type here
```suggestion
    return {};
```